easybuild/easyconfigs/b/BH/BH-1.60.0-1-foss-2015b-R-3.2.3.eb

```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Author: Adam Huffman <adam.huffman@crick.ac.uk>
+# The Francis Crick Institute
+
+easyblock = 'RPackage'
+
+name = 'BH'
+version = '1.60.0-1'
+
+homepage = 'http://cran.r-project.org/web/packages/%(name)s'
+description = """BH: Boost C++ Header Files for R"""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+source_urls = [
+    'http://cran.r-project.org/src/contrib/',
+    'http://cran.r-project.org/src/contrib/Archive/$(name)s/',
+]
+sources = ['%(name)s_%(version)s.tar.gz']
+
+r = 'R'
+rver = '3.2.3'
+versionsuffix = '-%s-%s' % (r, rver)
+
+dependencies = [
+    (r, rver),
+    ('Boost', '1.58.0', '-Python-2.7.10'),
```

Comment:
@verdurin: it's probably better not to use a Python-enabled version of Boost here, since it's only there for use in R?

Also, why 1.58.0? Does 1.60.0 work?